lib/widgets/molecules/multi_select_chips.dart

```diff
@@ -97,51 +153,49 @@ class CreateMultiSelectChips extends StatelessWidget {
   }
 }
 
-class FilterChipWidget extends StatefulWidget {
+class FilterChipWidget extends StatelessWidget {
+  final int id;
   final String chipName;
   final IconData? icon;
+  final bool selected;
+  final Function(int chipId) onSelectionChanged;
```

Comment:
Each chip should also store the backend textId. The chipName is what we show to users, but remember that those change depending on the user language so we should not use that string value as an identifier.

For example, content from the backend will have a format similar to this:

```
{
    textId: 'accountingAndFinance',
    <... some other fields...>
    translatedValue: 'Accounting and Finance',
}
```
The **translatedText** ('Accounting and Finance') changes depending on the user's language, but the **textId** remains the same so that is what we use to identify these values.

To make long story short, we need a an additional field `textId` in the FilterChipWidget, and then the helper functions will return those textIds instead of the chipNames. Let me know if you need more clarification on this!

Reply:
Hi Miguel, this is interesting! Thanks for pointing out. I have made the requested changes. PTAL